mods/tuxemon/l18n/en_US/LC_MESSAGES/base.po

```diff
@@ -1561,6 +1561,31 @@ msgstr "STOP"
 msgid "omnichannelarrives2"
 msgstr ""
 
+msgid "firsts"
+msgstr "testmebitch"
```

Comment:
Really?

Reply:
I don't see a problem with the wording: We can use more silly stuff like this... we're not bound by commercial standards after all :3 Granted it makes sense to have in the world, and it's not just adding random stuff for the sake of adding something (quality should be kept in mind).

Reply:
That was just me testing, I must have forgotten to take it out :P 
I was having an issue with the po files, but got it figured out

Reply:
in case you're interested the mom npc says it :D

Reply:
Tuxemon as a project should be strictly PG and inclusive.  I agree this should not included.